src/components/ResetPassword.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { Redirect } from "react-router-dom";
+import { handleResetPassword } from "../redux/actions/ResetPasswordAction";
+import SubmitButton from "../views/Form/SubmitButton";
+import { TextInput } from "../views/Form";
+
+export class ResetPassword extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: {
+        //   get token and email from the url query parameters
+        token: window.location.search
+          ? new URLSearchParams(window.location.search).getAll("token")[0]
+          : "",
+        email: window.location.search
+          ? new URLSearchParams(window.location.search).getAll("email")[0]
+          : "",
+        password: "",
+        confirm_password: ""
+      },
+      info: true
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
```

Comment:
you could refactor this to:
``` 
handleChange = e => {
....
}
````
so that you can get rid of the
```this.handleChange = this.handleChange.bind(this)``` 
in the constructor

Reply:
Do the same for ```handleSubmit```